src/main/java/Duke.java

```diff
@@ -1,10 +1,165 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import java.util.Scanner;
+
+
+public class Duke {
+    public static void printHorizontal () {
+        System.out.println("____________________________________________________________");
+    }
+    public static void printHeader() {
+        printHorizontal();
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        printHorizontal();
+    }
+    public static String extractKeyword (String command) {
+        String[] parsedStrings = command.split(" ", 2);
+        return parsedStrings[0];
+    }
+    public static String extractDetails (String command) {
+        String[] parsedStrings = command.split(" ", 2);
+        return parsedStrings[1];
+    }
+    public static boolean isBye (String keyword) {
+        return keyword.equals("bye");
+    }
+    public static int findCommandLength (String command) {
+        String[] parsedStrings = command.split(" ", 2);
+        return parsedStrings.length;
+    }
+    public static void printTask (Todo[] recordedList, int index) {
+        Todo element = recordedList[index];
+        System.out.print("[" + element.taskType + "]");
+        System.out.print("[" + element.getStatusIcon() + "] " + element.description);
+        System.out.println(element.getTimeline());
+    }
+    public static void printList (Todo[] recordedList, int listCount) {
+        if (listCount == 0) {
+            System.out.println("Sorry, you have no tasks in the list");
+        }
+        else {

```

Comment:
I believe the else (line 39) should be on the same line as the closing brace of the if statement (line 38)